tests/operators/test_constraints.py

```diff
@@ -16,9 +16,20 @@
 
 from datetime import datetime
 
+import iris
+import iris.cube
+import pytest
+
 from CSET.operators import constraints
 
 
+# Session scope fixtures, so the test data only has to be loaded once.
+@pytest.fixture(scope="session")
+def load_verticalcoord_cubelist() -> iris.cube.CubeList:
+    """Get a cubelist with multiple vertical level cubes."""
+    return iris.load("tests/test_data/vertlevtestdata.nc", "y_wind")
+
+
```

Comment:
```suggestion
```
To better isolate the functionality of the tests, perhaps we should just be testing the constraint looks like in this function, and then tests its actual filtering in test_filters.py?